### ISSUE TITLE
Support:  Problem with workers version misp2.5

### ISSUE THREAD

**Issue by sergirj-alt** [MISP/MISP#9957]
### Support Questions

Good afternoon, when updating the MISP, to version 2.5, the worker scheduler, has disappeared, I wanted to know if that is true or that worker is by default.

Greetings

![image](https://github.com/user-attachments/assets/9fee3540-def5-4b1f-b5ee-7be474280ed4)

### MISP version

2.5.0

### Operating System

ubuntu

### Operating System version

24.04

### PHP version

8.3

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by virasasi** [comment]
Yes, that is true.  The scheduler is removed from mine too.  I am running the current version of MISP 2.5.3
